src/test/java/com/redhat/cloud/notifications/processors/webhook/WebhookTest.java

```diff
@@ -76,14 +79,35 @@ void testWebhook() {
         webhookActionMessage.setApplication("WebhookTest");
         webhookActionMessage.setTimestamp(LocalDateTime.of(2020, 10, 3, 15, 22, 13, 25));
         webhookActionMessage.setEventType("testWebhook");
-
-        Map<String, Object> payload = new HashMap<>();
-        payload.put("any", "thing");
-        payload.put("we", 1);
-        payload.put("want", "here");
-        webhookActionMessage.setPayload(payload);
         webhookActionMessage.setAccountId("tenant");
 
+        Map<String, Object> payload1 = new HashMap<>();
```

Comment:
Did you had plans for `payload2` and then changed your mind? 😃

Reply:
Ah, yes. The request accepts multiple `payloads` in the events :)
payload2 is actually empty.